### PR TITLE
added paddding

### PR DESCRIPTION
**Pull request by ira-lapchuk**

### CODE REVIEW

File: style5.css [hunk 1]
```diff
@@ -19,17 +19,17 @@
...
-    /*padding-left: 50px;*/
     margin-bottom: 80px;
 }
 
 .row-1__text {
     display: flex;
     width: 100%;
     background-color: solid transparent;
-    border: 1px solid transparent;
+    border: 0px solid transparent;
```
Comment (lapc002) [inline style5.css]: why we need border then? we can remove it

File: style5.css [hunk 2]
```diff
@@ -111,27 +111,27 @@
...
     font-family: "Segoe UI", Roboto, "Helvetica Neue", "Noto Sans", "Liberation Sans", Arial, sans-serif;
-    padding-left: 50px;
 }
 
 .row-2__text {
     display: flex;
     width: 100%;
     background-color: solid transparent;
-    border: 2px solid transparent;
+    border: 0px solid transparent;
```
Comment (lapc002) [inline style5.css]: the same here

File: style5.css [hunk 3]
```diff
@@ -149,17 +149,19 @@
...
 }
 
-.row-2__title {
+.row-2__title h2 {
     display: flex;
     width: 100%;
     height: 180px;
     background-color: transparent;
-    border: 2px solid transparent;
+    border: 0px solid transparent;
```
Comment (lapc002) [inline style5.css]: and here

File: style5.css (hunk 1 above)
Comment (ira-lapchuk) [reply style5.css]: already done

File: style5.css (hunk 2 above)
Comment (ira-lapchuk) [reply style5.css]: and here

File: style5.css (hunk 3 above)
Comment (ira-lapchuk) [reply style5.css]: ok

**Review by lapc002 — APPROVED** [review]

**Pull request merged**